src/common/mixins/TrackerLayerMixin.js

```diff
@@ -575,32 +674,26 @@ const TrackerLayerMixin = (Base) =>
       return nextTick;
     }
 
+    /**
+     * Update filter provided by properties or permalink.
+     */
+    updateFilters() {
+      // Setting filters from the permalink if no values defined by the layer.
+      const parameters = qs.parse(window.location.search.toLowerCase());
```

Comment:
Do we really want to get `parameters` from `window.location`? IMHO this should only be moved to the app which is using `TrackerLayer` and passed down to `updateFilters()` as an argument.

Reply:
yes it's a discussion that I wanted to have with Uli about another ticket. I wait to know where it is used, and I will remove them in another PR.  The TrackerLayer provides a filter property where an app can provides whatever filter thex want, so this function seems useless anyway and could be removed